### PR TITLE
Minor clean up

### PR DESCRIPTION
**Pull request by bmarika**

### CODE REVIEW

File: fabfile.py [hunk 1]
```diff
@@ -103,18 +92,6 @@ def setup(*args, **kwargs):
     # needs to come first to as to index data as it is being loaded
     remove_search_index()
     create_search_index()
-    no_sudo = True if 'no-sudo' in args else False
```
Comment (ngash) [inline fabfile.py:106]: was this supposed to be removed or just to be done outside `fab setup` ?

File: fabfile.py [hunk 2]
```diff
@@ -44,17 +44,6 @@ def server_deploy():
...
 
-def reset_migrations():
-    """Development only - remove and recreate all migration"""
-    for app_name in base.LOCAL_APPS:
-        local('rm -f {}/migrations/ -r'.format(app_name))
-
-    for app_name in base.LOCAL_APPS:
-        manage('makemigrations {}'.format(app_name))
-
-    local('git add . --all')
```
Comment (ngash) [inline fabfile.py:55]: hii ilikuwa ukora

Comment (ngurenyaga) [inline fabfile.py:55]: Guilty as charged. It had its place and time - when it was me and @bmarika only working in the same room, shouting changes across the table.

That time is gone.

File: fabfile.py (hunk 1 above)
Comment (bmarika) [inline fabfile.py:106]: You have a point. A function with the db stuff would be better here

File: docs/02_developer_install.rst [hunk 3]
```diff
@@ -48,19 +48,19 @@ Getting started
...
 
-3. Run ``fab setup``
-    Running this command will do the following:
-
-    * Create the database.
-
-    * Run migrations.
-
-    * Load demo data if the project **DEBUG** attribute in settings is set to true .
+3. Run the following commands sequentially:
```
Comment (ngash) [inline docs/02_developer_install.rst:51]: Separate the setup install commands into:
1) Setup with existing database
2) Setup without existing database 
3) Recreating the database

**Pull request merged**